FreeRTOS/FreeRTOS-Kernel
# Upgrade msvc port to winsock2

Description
-----------
MBedTLS swaps to using WinSock2.h in their files after the V3.3.0 Version.
Upgrade the MSVC port to use WinSock2.h inorder to be compatible with future mbedtls releases.

Test Steps
-----------
Ran tests in FreeRTOS/FreeRTOS

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
